gradio/inputs.py

```diff
@@ -43,17 +41,17 @@ def __init__(
             DeprecationWarning,
         )
         super().__init__(
+            default_value=default,
```

Comment:
**Important**: in all of the components in `inputs.py` and `outputs.py`, the parameter should still be called `default` for backwards compatibility.

Reply:
Well it is default in the inputs.py, it is super init here

Reply:
Oh you are right my bad, GitHub's code preview skipped over a few lines :)